src/dlgAboutDialog.cpp

```diff
@@ -35,6 +42,8 @@
 #include <QDebug>
 #include "post_guard.h"
 
+const QString QUESTION_MARK = QStringLiteral("?");
```

Comment:
This is a bit over the top, we don't need a constant for just 2 usages. Let's not get crazy with these constants that make code readability more difficult.

Reply:
Also it is not going to be helpful for Linux distributions (or other OSes, like FreeBSD) where the project is built from our source tarball rather than a git checkout - they will not want "?" showing up and instead will probably have to take extra effort to prepare a local patch to cut out the code; which will not be encouraging for them...

\<ASIDE>{c.f. the different splash screens which we/I have take care to ensure they can disable - with a `"WITH_VARIABLE_SPLASHSCREEN=NO` so that a build which they add their own build designation to as a `MUDLET_VERSION_BUILD=something` keeps the default *release* branding!}\</ASIDE>